operator/v1/types_console.go

```diff
@@ -143,8 +143,95 @@ type Capability struct {
 	Visibility CapabilityVisibility `json:"visibility"`
 }
 
+// ThemeType is the value of the CustomLogo Theme Type field that determines if the theme is for the dark or light mode in the console UI.
+type ThemeType string
+
+// ThemeType values
+const (
+	// ThemeTypeDark represents the dark theme for the console UI.
+	ThemeTypeDark ThemeType = "Dark"
+
+	// ThemeTypeLight represents the light theme for the console UI.
+	ThemeTypeLight ThemeType = "Light"
+)
+
+// LogoType is the value of the CustomLogo Type field that determines if the logo is for the masthead or the favicon in the console UI.
+type LogoType string
+
+const (
+	// Masthead represents the logo in the masthead.
+	LogoTypeMasthead LogoType = "Masthead"
+
+	// Favicon represents the favicon logo.
+	LogoTypeFavicon LogoType = "Favicon"
+)
+
+// Theme defines a theme type for the console UI.
+type Theme struct {
+	// type is a required field that specifies the type of the logo for the console UI.
+	// This field is an enum with valid values "Dark" and "Light".
+	// The type determines whether the logo is for the dark or light theme.
+	// When set to "Dark", the corresponding logo for the dark theme will be used in the console UI.
+	// When set to "Light", the corresponding logo for the light theme will be used in the console UI.
+	// +kubebuilder:validation:Enum="Dark";"Light"
```

Comment:
I think this should be specified as part of the type

Reply:
I don't know that we have a clear opinion on defining as part of the type versus the field, just that it has to be on one or the other (not both). Whatever approach you prefer is fine.

Reply:
I thought it the validation should be tied to the type. If its not causing any issues, im fine leaving it here as well 👍